src/pages/Components/Components.tsx

```diff
@@ -214,6 +215,14 @@ const Test: FC = () => {
           iconPositioning={IconPositioningTypes.Right}
         />
       </DynamicForm>
+      <ProgressBar
```

Comment:
**[non-blocker]**

Not too important and you can decide this, when creating the actual modal, where this component is used, but gut feeling is that it would be more appropriate to use something like "activeStep" prop here. As far as I know it shouldn't be possible to have multiple active steps + it will probably be easier to use if we just change the "activeStep" prop, instead of mapping over the list of all steps every time we want to change the active step.